adafruit_ads1x15/adafruit_ads1x15.py

```diff
@@ -131,6 +130,23 @@ def __init__(self, i2c, address=ADS1X15_DEFAULT_ADDRESS):
                           ADC_Channel(self, 2),
                           ADC_Channel(self, 3)]
 
+    def _write_register(self, reg, value):
+        """Write 16 bit value to register."""
+        self.buf[0] = reg
+        self.buf[1] = (value >> 8) & 0xFF
+        self.buf[2] = value & 0xFF
+        with self.i2c_device as i2c:
+            i2c.write(self.buf)
+
+    def _read_register(self, reg):
+        """Return 16 bit register value as tuple of (LSB, MSB)."""
+        self.buf[0] = reg
+        with self.i2c_device as i2c:
+            i2c.write(self.buf, end=1, stop=False)
+            i2c.readinto(self.buf, start=1)
+        #         LSB          MSB
+        return self.buf[2], self.buf[1]
```

Comment:
This function is only used in one place, does it really make sense to have it?
Also, where it is used, we don't use the first byte of the result, does it really make sense to create a new tuple every time, especially since we do it in a tight loop?

Reply:
Yah, I was wondering about the tuple also. Basically it's just left over from first version of code. I was thinking of changing to just passing back an int. I'll take a look, I'm not sure why I can't just go ahead and do that. 

I think having the function makes sense for any future expansion of functionality - it'll be the generic register read for whatever.